soccer/src/soccer/strategy/coach/coach_node.hpp

```diff
@@ -77,10 +83,13 @@ class CoachNode : public rclcpp::Node {
     rj_msgs::msg::FieldDimensions current_field_dimensions_;
     bool have_field_dimensions_ = false;
 
+    int goalie_id_ = 0;  // sensible default?
```

Comment:
```suggestion
    int goalie_id_{0};
```
yes (but prefer brace init)